src/containers/global.types.ts

```diff
@@ -1,3 +1,8 @@
+/**
+ * The types that represents the contract of the global UI state.
+ *
+ * Should be used on inversify hook to get a instance of the global UI state
+ */
```

Comment:
I guess we can omit this from bundle cause we can provide some hooks like `useUIStore()`
So, no need for JSDocs here :)